<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>McrPlotWidget</class>
 <widget class="QWidget" name="McrPlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="CaptionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ConcentrationSpectralLabelLaout">
     <item>
      <widget class="QLabel" name="ConcentrationPlotLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Concentration profilies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SpectalPlotLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Spectral profiles</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ConcentrationSpectralPlotLayout">
     <item>
      <widget class="Widgets::MCRCalculationPlot" name="ConcentrationPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Widgets::MCRCalculationPlot" name="SpectralPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="DifferenceOriginalrestoredLabelLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="DeifferenceButtonLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="DifferenceLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Difference plot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelExcludeFromDifferenceButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Cancel select from exclude</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="OriginRestoredButtonLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="OriginalRestoredLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Original restored</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelExcludeFromOriginRestoreButton">
         <property name="text">
          <string>Cancel select from exclude</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="DifferenceOriginPlotLayout">
     <item>
      <widget class="Widgets::McrDifferencePlot" name="DifferencePlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Widgets::McrOriginRestoredPlot" name="OriginalRestoredPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="legendLabel">
     <property name="text">
      <string>Legend</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="legendGridLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::MCRCalculationPlot</class>
   <extends>QWidget</extends>
   <header>ui/widgets/mcr_calculation_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widgets::McrDifferencePlot</class>
   <extends>QWidget</extends>
   <header>ui/widgets/mcr_difference_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widgets::McrOriginRestoredPlot</class>
   <extends>QWidget</extends>
   <header>ui/widgets/mcr_origin_restored_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
